content/GettingStarted/Migrate/MigratePrep.md

```diff
@@ -77,13 +77,10 @@ CORE Enterprise customers are encouraged to contact Support for assistance with
       {{< include file="/static/includes/NetworkInstallRequirementsSCALE.md" >}}
 
 6. Migrate the deprecated S3 MinIO service (if in use). See [services deprecated in SCALE](#migrating-from-deprecated-services).
-   This is a lenthy process depending on the amount of data stored while using the S3 service.
-   Read and follow instructions in [Migrating from MinIO S3](https://www.truenas.com/docs/scale/22.12/scaletutorials/apps/communityapps/minioclustersscale/migratingfroms3service/)!
-   Make sure S3 MinIO data is backed up as a precaution. The migrating from the S3 service requires installing SCALE 22.12.3 (Bluefin).
-   This version of SCALE provides access to both the S3 service and the MinIO app you migrate to.
+   This is a lengthy process depending on the amount of data stored while using the S3 service.
+   Read and follow instructions in [Migrating MinIO Data from CORE to SCALE](https://www.truenas.com/docs/solutions/miniocoretoscale/)!
 
-   If you need to do a clean install with the SCALE <file>iso</file> file, you can import your data pools into SCALE.
-   Verify you can access your backed up files before you upgrade/migrate to SCALE.
+   Make sure S3 MinIO data is backed up as a precaution. The migration process from the S3 service requires first [migrating to the MinIO plugin in TrueNAS CORE](https://www.truenas.com/docs/core/13.0/coretutorials/jailspluginsvms/plugins/minioplugin/#migrating-from-s3-service-to-minio-plugin), migrating from CORE to SCALE, then installing the SCALE MinIO app and importing S3 data.
```

Comment:
Actually, this is not true. You can upgrade to the CORE plugin or you can upgrade to Bluefin app. They can use either method. In the conversation with Chris last week he said they can use either method. We should provide context and both methods as options for user. If they use the CORE plugin, when they migrate to SCALE they have to set up MinIO twice. If they use the Bluefin migration method they only set up or configure MinIO once so some users might prefer that over S3 to CORE config then CORE to SCALE app config.